ignition-server/controllers/local_ignitionprovider.go

```diff
@@ -268,15 +269,56 @@ func (p *LocalIgnitionProvider) GetPayload(ctx context.Context, releaseImage str
 	err = func() error {
 		start := time.Now()
 		binaries := []string{"machine-config-operator", "machine-config-controller", "machine-config-server"}
+		suffix := ""
+
+		mcoOSReleaseBuf := &bytes.Buffer{}
```

Comment:
can we please follow up to unit test this?

Reply:
just to be clear, unit test the extraction of the RHEL major version from `/etc/os-release`?